### PR TITLE
start nodes before run integration test

### PR DESCRIPTION
**Pull request by giogio21**
1. Start and set up nodes in CI to allow scenario_test.go to work
2. Change timeout integration_test.go
3. Add new tmux window when we run nodes
4. Update readme
5. Add env variable to scenario_test.go

### CODE REVIEW

**Review by nkcr — COMMENTED** [review]
Nice!

File: integration/scenario_test.go [hunk 1]
```diff
@@ -33,7 +34,16 @@ var suite = suites.MustFind("Ed25519")
...
+	val, ok := os.LookupEnv("DVOTING_NB_NODE")
+	var numNodes int
+	var err error
+	if !ok {
+		numNodes = 3
+	} else {
+		numNodes, err = strconv.Atoi(val)
+		require.NoError(t, err, "Env var DVOTING_NB_NODE is not configurated")
+	}
+	t.Run("Basic configuration", getScenarioTest(numNodes, 3, 1))
```
[the remark this answers is not in the document]
Comment (nkcr) [reply integration/scenario_test.go]: is it really resolved ..? ;)

File: README.md [hunk 2]
```diff
@@ -147,26 +152,33 @@ rm -rf /tmp/node{1,2,3}
...
 scenario:
 
 ```sh
 sk=28912721dfd507e198b31602fb67824856eb5a674c021d49fdccbe52f0234409
 LLVL=info memcoin --config /tmp/node1 e-voting scenarioTest --secretkey $sk
 ```
 
+You can also run scenario_test.go, by running in the integration folder this command:
+```sh
+DVOTING_NB_NODE=3 go test -v scenario_test.go
```
[the remark this answers is not in the document]
Comment (nkcr) [reply README.md]: Note the double N. We already have LLVL for Log Level, so NNODE for Num Nodes.

File: integration/scenario_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (giogio21) [reply integration/scenario_test.go]: Do you talk about the second arg of getScenarioTest ? Maybe I can add a second environment variable so we can also choose the number of ballot casted or it is enough to have 1 ballot cast for each node ?

[the remark this answers is not in the document]
Comment (nkcr) [reply integration/scenario_test.go]: sorry for not being clear, my comment was making a suggestion to update your code with my proposal.

**Review by nkcr — APPROVED** [review]
LGTM 🚀

**Pull request merged**